sw/applications/src/kernels/batchnorm.c

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "snrt.h"
+
+/**
+ * @brief implementation of a FP64 batchnorm as a linear combination
+ * y = gamma * x + beta
+ *
+ * @param ifmap pointer to input feature map
+ * @param gamma pointer to gamma
+ * @param beta pointer to beta
+ * @param ofmap pointer to output feature map
+ * @param OW width of output feature map
+ * @param CI number of input channels
+ * @param compute_num number of compute units
+ * @param setup_SSR setup SSR strides and bounds
+ */
```

Comment:
Duplicate docstring: If the header is documented we can omit this here and leave only in-line comments to document the code.